pkgs/by-name/bi/binary-object-scanner/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+}:
+
+buildDotnetModule rec {
+  pname = "BinaryObjectScanner";
+  version = "3.3.1";
+
+  src = fetchFromGitHub {
+    owner = "SabreTools";
+    repo = "BinaryObjectScanner";
+    rev = "3.3.1";
+    hash = "sha256-rEtWJ87AQOrEXe4k6geWZ1pAOgC/HnPe7iJ5xVyMC1M=";
+  };
+
+  dotnet-sdk = dotnetCorePackages.sdk_9_0;
+  dotnet-runtime = dotnetCorePackages.runtime_9_0;
+  dotnetFlags = [ "-p:TargetFramework=net9.0" ];
+  nugetDeps = ./deps.nix;
+
+  projectFile = [
+    "ProtectionScan/ProtectionScan.csproj"
+    "ExtractionTool/ExtractionTool.csproj"
+  ];
+  executables = [
+    "ProtectionScan"
+    "ExtractionTool"
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/SabreTools/BinaryObjectScanner";
+    description = "C# protection, packer, and archive scanning library. Provides ProtectionScan and ExtractionTool.";
+    license = licenses.mit;
+    maintainers = with maintainers; [ hughobrien ];
+    platforms = [ "x86_64-linux" ];
+    mainProgram = "ProtectionScan";
+  };
```

Comment:
```suggestion
  meta = {
    homepage = "https://github.com/SabreTools/BinaryObjectScanner";
    description = "C# protection, packer, and archive scanning library. Provides ProtectionScan and ExtractionTool.";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ hughobrien ];
    platforms = lib.platforms.unix ++ lib.platforms.windows;
    mainProgram = "ProtectionScan";
  };
```

Reply:
If you just want this to support all the platforms that the dotnet-sdk/runtime supports, you can just leave it blank and `buildDotnetModule` will fill it in.  It's going to intersect it with the sdk platforms anyway.